src/Batch/Client/changelog.md

```diff
@@ -1,16 +1,23 @@
 ## Azure.Batch release notes
 
+### Changes in 6.1.0
+#### REST API version
+This version of the Batch .NET client library targets version 2017-01-01.4.0 of the Azure Batch REST API.
+
+#### Packaging
+- In addition to a `net45` assembly, the package now includes a `netstandard1.5` assembly. The `net45` assembly is identical to the one from `6.0.0`.
```

Comment:
Is this meant to communicate "the client library is now supported on .NET Core"?  If so perhaps say that up front, and talk about the physical aspects as a way of explaining how and which versions.  .NET nomenclature is so confusing that talking about `netstandard1.5` is about as meaningful (at least to _this_ developer) as writing our change logs in Ancient Greek!